.github/bin/minikube/provision-cluster.sh

```diff
@@ -9,6 +9,7 @@
 #
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
+set -ex
 
 # Minikube environments config
 export MINIKUBE_VERSION=v1.8.2
```

Comment:
Is it too old ?

Reply:
I'm changing the minikube version in stable tests.

Reply:
It will be a separate PR